### DISCUSSION TITLE
External config in json

### DISCUSSION THREAD

**Discussion by tam481** [t0bst4r/home-assistant-matter-hub#113] [General]
Hello
Do you have any plans to support all parameters to be in the json file like Matterbridge?

Parameters such as HAMH_HOME_ASSISTANT_URL etc

**Comment by t0bst4r** [comment]
More or less 😅
They are not specific per bridge, so it doesn’t make sense to put it there.
Anyway I am using `yargs` to parse cli commands and env variables. I could activate the `config` property there, which allows having an **additional** config file for value which are provided as cli or env at the moment.

**Comment by t0bst4r** [comment]
I’ve created an issue to track progress.
